docs/_docs/integration-checks-reference.md

```diff
@@ -927,16 +935,23 @@ See [Sample Data](./faq#output) in the FAQ for more details on this.
 
 ## T006 - Polling Sample Contains a Subset of Keys from Live Result
 
-For hook triggers, we require you to provide a Perform List URL so that users can
-pull a live sample in the Zap editor. This is called a Polling Sample, and is created
+For REST Hook triggers, we require you to provide a Perform List URL (check `D006`) so that users can
+retrieve a real data sample in the Zap editor. This is called a polling sample, and is created
 when you test the trigger in a Zap before turning it on.
```

Comment:
we call out the Zap editor above, so maybe "test the trigger in the Zap editor before turning it on"

Reply:
that sounds good!